docs/tutorials/mpv.md

```diff
@@ -238,6 +238,10 @@ deband-threshold=48
 deband-grain=48
 ```
 
+!!!warning
+Your deband settings should be placed after your [`profile`](https://mpv.io/manual/stable/#profiles) or at the end of the file.
```

Comment:
Putting it at the end will then cause conflicts with user made profiles. Ideally:
```
<built in profile>
 gpu-hq 
<user settings>
deband = no 
<user profiles>
e.g simulcast profile that turns on deband
```

Reply:
Proposed change in https://github.com/Snaacky/thewiki/pull/150/commits/323dda8f959e30bff6bb862029b45010d0959430.